client/i18n/locales/english/translations.json

```diff
@@ -510,6 +510,28 @@
       "learn-more": "Learn more about <0>Gitpod workspaces.</0>",
       "logout-warning": "If you log out of freeCodeCamp before you complete the entire {{course}} course, your progress will not be saved to your freeCodeCamp account."
     },
+    "local": {
+      "intro": "This course runs in a virtual Linux machine on your computer. To run the course, you first need to download each of the following if you don't already have them:",
+      "download-vscode": "<0>VS Code</0> and the <1>Dev Containers</1> extension",
+      "heading": "Then, follow these instructions to start the course:",
+      "step-1": "Open a terminal and clone the RDB Alpha repo if you don't already have it with <0>git clone https://github.com/freeCodeCamp/rdb-alpha</0>",
+      "step-2": "Navigate to the <0>rdb-alpha</0> directory in the terminal with <1>cd rdb-alpha</1>, and open VS Code with <2>code .</2>",
```

Comment:
At this point the dev containers extension creates a pop up

> Folder contains a Dev Container configuration file. Reopen folder to develop in a container ([learn more](https://aka.ms/vscode-remote/docker)).

We should probably explain what to do about that

Reply:
I didn't actually see this. Perhaps cause I already opened the repo and/or maybe clicked the "x" or "don't show again" or something. I would like to keep these instructions as concise as possible, so adding something like "If you see a pop-up in the corner, you can click that to open the project in a container." I think I would prefer to just keep one method for opening things in the instructions. If people notice the pop-up, they can close it or open it like that. Although, they wouldn't want to open it right away if they're saving their progress.

Reply:
There's already a lot of info to convey, so, sure, let's ignore the popup.